salt/netapi/rest_cherrypy/app.py

```diff
@@ -2326,7 +2326,6 @@ def signal_handler(signal, frame):
                         logger.error(
                                 "Error: Salt event has non UTF-8 data:\n{0}"
                                 .format(data))
-                time.sleep(0.1)
```

Comment:
@pass-by-value we are wondering why you would add that `sleep` here. :wink: Any insights?